src/csharp/Microsoft.Spark.E2ETest/UdfTests/UdfComplexTypesTests.cs

```diff
@@ -295,6 +295,27 @@ public void TestUdfWithReturnAsRowType()
                         outerCol.GetAs<Row>("col3"));
                 }
             }
+
+            // Chained UDFs.
+            {
+                var schema = new StructType(new[]
+                {
+                    new StructField("col1", new IntegerType()),
+                    new StructField("col2", new StringType())
+                });
+                Func<Column, Column> udf1 = Udf<string>(
+                    str => new GenericRow(new object[] { 1, "abc" }), schema);
+
+                Func<Column, Column> udf2 = Udf<Row, string>(
+                    row => row.GetAs<string>(1));
+
+                Row[] rows = _df.Select(udf2(udf1(_df["name"]))).Collect().ToArray();
+                Assert.Equal(3, rows.Length);
+
+                var expected = new[] { "abc", "abc", "abc" };

```

Comment:
Can we update the UDF to relay the name?
For example,
```C#
Func<Column, Column> udf1 = Udf<string>(
    str => new GenericRow(new object[] { 1, str }), schema);
```

Reply:
Sure, changed.